[?1049h[1;24r(B[m[4l[?7h[?1l>Traceback (most recent call last):
  File "/usr/local/bin/deluge-console", line 9, in <module>
    load_entry_point('deluge==1.3.15', 'console_scripts', 'deluge-console')()
  File "/usr/local/lib/python2.7/dist-packages/deluge-1.3.15-py2.7.egg/deluge/ui/console/main.py", line 75, in start
    Console().start()
  File "/usr/local/lib/python2.7/dist-packages/deluge-1.3.15-py2.7.egg/deluge/ui/console/main.py", line 72, in start
    ConsoleUI(self.args)
  File "/usr/local/lib/python2.7/dist-packages/deluge-1.3.15-py2.7.egg/deluge/ui/console/main.py", line 209, in __init__
    curses.wrapper(self.run)
  File "/usr/lib/python2.7/curses/wrapper.py", line 49, in wrapper
    curses.nocbreak()
_curses.error: nocbreak() returned ERR
